source/server-selection/server-selection.rst

```diff
@@ -730,33 +730,35 @@ the command and how it is invoked:
 
       The current list of "may-use-secondary" commands includes:
 
-        - aggregate without a write stage (e.g. ``$out``, ``$merge``)
-        - collStats
-        - count
-        - dbStats
-        - distinct
-        - find
-        - geoNear
-        - geoSearch
-        - group
-        - mapReduce where the ``out`` option is ``{ inline: 1 }``
-        - parallelCollectionScan
+      - aggregate without a write stage (e.g. ``$out``, ``$merge``)
+      - collStats
+      - count
+      - dbStats
+      - distinct
+      - find
+      - geoNear
+      - geoSearch
+      - group
+      - mapReduce where the ``out`` option is ``{ inline: 1 }``
+      - parallelCollectionScan
```

Comment:
Just fixing indentation here. Also fixed some grammar and capitalization (in the mapReduce paragraph) below.